merlin/systems/dag/runtimes/base_runtime.py

```diff
@@ -33,9 +34,32 @@ def __init__(self, executor=None):
             The Graph Executor to use to use for the transform, by default None
         """
         self.executor = executor or LocalExecutor()
-        self.op_table = {}
+        self.op_table = OpTable()
 
-    def transform(self, graph: Graph, transformable: Transformable):
+    def convert(self, graph: Graph):
+        """
+        Replace the operators in the supplied graph with ops from this runtime's op table
+
+        Parameters
+        ----------
+        graph : Graph
+            Graph of nodes container operator chains for data manipulation.
+
+        Returns
+        -------
+        Graph
+            Copy of the graph with operators converted to this runtime's versions
+        """
+        if self.op_table:
```

Comment:
if `self.op_table` is always an instance of `OpTable` this contition might always be true

Reply:
Ah, good catch. A plain dictionary would evaluate as false-y if it was empty here. 👍🏻